spec/services/events/stores/clickhouse_store_spec.rb

```diff
@@ -51,6 +60,10 @@
         end
       end
 
+      if i.zero?
```

Comment:
Style/IfUnlessModifier: Favor modifier `if` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`. (https://rubystyle.guide#if-as-a-modifier)